netadapter/server/grpcserver/protowire/message_request_relay_blocks.go

```diff
@@ -0,0 +1,30 @@
+package protowire
+
+import (
+	"github.com/kaspanet/kaspad/wire"
+	"github.com/pkg/errors"
+)
+
+func (x *KaspadMessage_RequestRelayBlocks) toWireMessage() (wire.Message, error) {
+	if len(x.RequestRelayBlocks.Hashes) > wire.MsgGetRelayBlocksHashes {
```

Comment:
Please rename `wire.MsgGetRelayBlocksHashes` to `MaxMsgRequestBlockHashes` or something (Also go over the other renamings you did and see if there are any similar errors. I'll check myself, but might miss something.